integration-auth/src/main/java/org/symphonyoss/integration/authentication/jwt/JwtAuthentication.java

```diff
@@ -33,10 +50,20 @@
 
   public static final String AUTHORIZATION_HEADER_PREFIX = "Bearer ";
 
+  private static final int CACHE_EXPIRATION_IN_MINUTES = 60;
```

Comment:
Well, you have told me that the POD public certificate rarely changes, what would be a good number?